t/05-scheduler-full.t

```diff
@@ -215,21 +179,19 @@ subtest 'Simulation of unstable workers' => sub {
 
 subtest 'Simulation of heavy unstable load' => sub {
     my $allocated;
-    my $failures;
-    my $no_actions;
     my @workers;
     dead_workers($schema);
     my @duplicated;
 
     push(@duplicated, $_->auto_duplicate()) for $schema->resultset("Jobs")->latest_jobs;
 
     push(@workers, unresponsive_worker($k->key, $k->secret, "http://localhost:$mojoport", $_)) for (1 .. 50);
-    sleep 5;
+    my $i = 4;
+    wait_for_worker($schema, ++$i) for 1 .. 10;
 
-    ($allocated, $failures, $no_actions) = scheduler_step($reactor); # Will try to allocate to previous worker and fail!
-    is @$allocated, 10, "Allocated maximum number of jobs that could have been allocated";
-    is $failures, 2, "Failure count should be to 2, since we took too much time to schedule";
-    is get_scheduler_tick($reactor), 2**$failures * $ENV{OPENQA_SCHEDULER_TIMESLOT}, "Tick is at the expected value";
+    ($allocated) = scheduler_step($reactor);    # Will try to allocate to previous worker and fail!
+    is @$allocated, 8,
```

Comment:
This is not clear to me.. why before we could allocate 10, and now 8?

Reply:
See https://github.com/os-autoinst/openQA/pull/1776